internal/output/output.go

```diff
@@ -92,81 +92,96 @@ type Output struct {
 
 // SetImageAccessibleCheck sets the passed and result.message fields of the ImageAccessibleCheck to the given values.
 func (o *Output) SetImageAccessibleCheckFromError(err error) {
+	metadata := map[string]interface{}{
+		"code":        "image.accessible",
+		"collections": []string{"minimal"},
```

Comment:
Thinking about collection minimal here...

These rules are a bit special since they cannot be skipped, so we maybe have to think about this a little.

E.g. consider if a user does `exclude: '@minimal'` then (correctly) expects this to be skipped.

Reply:
I don't think we should add "collections" here.  Maybe we can set the code to indicate this is some sort of built-in thing?

Reply:
Maybe something like `builtin.image_accessible`?

Reply:
I like `builtin`.